edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -67,6 +67,14 @@ def authenticate(self, request):
             if not user_and_auth:
                 return user_and_auth
 
+            # Fail authentication if user disabled
+            user = user_and_auth[0]
+            if user and isinstance(user, get_user_model()):
+                if not user.has_usable_password():
+                    msg = 'User is disabled.'
+                    logger.exception(msg)
```

Comment:
This would be an exception because the user is being denied entry. I believe the info may be used to log data while user is following expected flow of use.

Reply:
1. Would you mind making the log message more descriptive? Something like:
```
User id XXX attempted JWT authentication after being disabled by an admin.
```
2. I think an error log would be fine, since we don't really need the stacktrace. Hopefully my last note on this. :)